src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/BaseIntegrationTest.java

```diff
@@ -39,6 +39,7 @@
     protected static final String JURISDICTION_ID = "CMC";
     protected static final String CASE_TYPE_ID = "MoneyClaimCase";
     protected static final String CREATE_NEW_CASE = "IssueClaim";
+    protected static final String LINK_LETTER_HOLDER = "LinkLetterHolder";
```

Comment:
Dont recreate the event name. Please use the CaseEvent enum.

Reply:
good pick, done